examples/llm_stack/README.md

```diff
@@ -0,0 +1,12 @@
+# Evaluate components of your LLM stack
```

Comment:
```suggestion
# Flexibly change components of your LLM stack
```

Reply:
Can you add the DAG image to the README too please?

Reply:
Done!